client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker-preview.tsx

```diff
@@ -607,22 +608,32 @@ const UnifiedDesignPickerPreview = ( {
 				/>
 			) }
 			<QueryEligibility siteId={ site?.ID } />
-			<EligibilityWarningsModal
-				site={ site ?? undefined }
-				isMarketplace={ selectedDesign?.is_externally_managed }
-				isOpen={ showEligibility }
-				handleClose={ () => {
-					recordTracksEvent( 'calypso_automated_transfer_eligibility_modal_dismiss', {
-						flow: 'onboarding',
-						theme: selectedDesign?.slug,
-					} );
-					setShowEligibility( false );
-				} }
-				handleContinue={ () => {
-					navigateToCheckout();
-					setShowEligibility( false );
-				} }
-			/>
+			{ showEligibility && (
+				<Modal
+					className="eligibility-warnings-modal__dialog-content"
+					title={ translate( 'Before you continue' ) }
+					onRequestClose={ () => {
+						recordTracksEvent( 'calypso_automated_transfer_eligibility_modal_dismiss', {
+							flow: 'onboarding',
+							theme: selectedDesign?.slug,
+						} );
+						setShowEligibility( false );
+					} }
+					size="medium"
+				>
+					<EligibilityWarnings
+						siteId={ site?.ID }
+						standaloneProceed
+						isOnboarding
+						isMarketplace={ selectedDesign?.is_externally_managed }
+						currentContext="plugin-details"
```

Comment:
I don't love that "plugin-details" is used as a shorthand for "business plan that isn't atomic yet". Might be worth renaming to something more generic.

Reply:
Agreed, I found it confusing that my help redirection change was being applied in non plugin contexts.